src/test/java/org/opentripplanner/openstreetmap/tagmapping/HamburgMapperTest.java

```diff
@@ -0,0 +1,48 @@
+package org.opentripplanner.openstreetmap.tagmapping;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.opentripplanner.openstreetmap.model.OSMWithTags;
+
+public class HamburgMapperTest {
+
+  private HamburgMapper mapper;
+
+  @BeforeEach
+  void createMapper() {
+    mapper = new HamburgMapper();
+  }
+
+  @Test
+  public void shouldAllowThroughTraffic_WhenAccessCustomers_AndCustomersHVV() {
+    OSMWithTags way = new OSMWithTags();
+    way.addTag("access", "customers");
+    way.addTag("customers", "HVV");
+
+    boolean generalNoThroughTraffic = mapper.isGeneralNoThroughTraffic(way);
+
+    assertFalse(
+      generalNoThroughTraffic,
+      "access=customers and customers=hvv should not be considered through-traffic"
```

Comment:
or the double negative "not be no-through traffic" but that is a bit of mind-bender.

Reply:
My reasoning is this: The method checks if the way generally disallows through-traffic. That behaviour was modified so that the way isn't preventing through-traffic if the combination of access=customers and customers=HVV is present. 

If the test assertions fails, the way prevents through-traffic even in the case of the combination. Therefore, the error message shown should be stating the expected situation and that is: the way does not prevent through-traffic if this combination is present.

I can improve the wording though to make that more clear.

Reply:
No, it's fine. Let's not split hairs over something that isn't very important.